server/sql/migrations/00002_init_schema_tables.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS users (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  name TEXT NOT NULL,
+  created TIMESTAMPTZ DEFAULT NOW()
+);
+
+CREATE TABLE IF NOT EXISTS subscriptions (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  name TEXT NOT NULL,
+  nickname TEXT,
+  created_at TIMESTAMPTZ DEFAULT NOW(),
+  removed_at TIMESTAMPTZ,
```

Comment:
either change these to `created` and `removed` to match the `users` table, or change the `users` table column `created_at` to match the `subscriptions` table. we should be consistent with table column naming